Gemfile.lock

```diff
@@ -3,88 +3,88 @@ GEM
   specs:
     action_policy (0.6.5)
       ruby-next-core (>= 0.14.0)
-    actioncable (7.0.4.3)
-      actionpack (= 7.0.4.3)
-      activesupport (= 7.0.4.3)
+    actioncable (7.0.5)
```

Comment:
I'm skeptical about the changes in this lock file. Please first do merge/rebase from master and then regenerate this file cleanly.